readthedocs/doc_builder/director.py

```diff
@@ -196,9 +195,8 @@ def build(self):
         self.run_build_job("post_build")
         self.store_readthedocs_build_yaml()
 
-        if self.data.project.has_feature(Feature.DISABLE_SPHINX_MANIPULATION):
-            # Mark this version to inject the new js client when serving it via El Proxito
-            self.data.version.addons = True
+        # Mark this version to inject the new js client when serving it via El Proxito.
+        self.data.version.addons = True
```

Comment:
We can delete the field `Version.addons` now. We are not using it anymore since we are always inject addons unless `AddonsConfig.enabled=False` -- but we are not checking for `Version.addons`.